dev-games/simgear/simgear-2018.2.2.recipe

```diff
@@ -0,0 +1,95 @@
+SUMMARY="Simulator Construction Toolkit"
+DESCRIPTION="Set of open-source libraries designed as building blocks \
+for quickly assembling 3D simulations, games, and visualization applications."
```

Comment:
Maybe:
```diff
-SUMMARY="Simulator Construction Toolkit"
-DESCRIPTION="Set of open-source libraries designed as building blocks \
-for quickly assembling 3D simulations, games, and visualization applications."
+SUMMARY="A Simulator Construction Toolkit"
+DESCRIPTION="SimGear is a set of open-source libraries designed as building \
+blocks for quickly assembling 3D simulations, games, and visualization \
+applications."
```

Reply:
That's because:
1) we prefer to have a "A ..." in `SUMMARY`.
2) it's nice to mention the name of the project somewhere near the beginning of `DESCRIPTION`.